src/ql/ir/old_to_new.cc

```diff
@@ -460,17 +467,27 @@ Ref convert_old_to_new(const compat::PlatformRef &old) {
 
                 // Check whether the operand list matches the specializations
                 // specified in the old way.
-                if (insn->template_operands.size() > insn->operand_types.size()) {
+                // FIXME: debug WIP, also see use of template_operands further down
```

Comment:
Can you explain more on this? What is the problem? I guess you have a use case!

Reply:
I'll clean this up, it was added while debugging